src/database/sql_database.cc

```diff
@@ -1741,11 +1756,7 @@ std::unique_ptr<Database::ChangedContainers> SQLDatabase::_recursiveRemove(
 
 std::unique_ptr<Database::ChangedContainers> SQLDatabase::_purgeEmptyContainers(const std::unique_ptr<ChangedContainers>& maybeEmpty)
 {
-#if SPDLOG_VERSION >= 10802
-    log_debug("start upnp: {}; ui: {}", fmt::join(maybeEmpty->upnp, ","), fmt::join(maybeEmpty->ui, ","));
-#else
-    log_debug("start upnp: {}; ui: {}", fmt::format("{}", fmt::join(maybeEmpty->upnp, ",")), fmt::format("{}", fmt::join(maybeEmpty->ui, ",")));
-#endif
+    log_debug("start upnp: {}; ui: {}", fmt::to_string(fmt::join(maybeEmpty->upnp, ",")), fmt::to_string(fmt::join(maybeEmpty->ui, ",")));
```

Comment:
I'm quite sure, you don't intend to revert that change?

Reply:
I came across the same pattern in the last commit. I think using fmt::to_string is readable enough to leave it like this. So I removed the #if